ui/components/app/modals/confirm-remove-account/confirm-remove-account.component.js

```diff
@@ -47,7 +38,9 @@ export default class ConfirmRemoveAccount extends Component {
 
   renderSelectedAccount() {
     const { t } = this.context;
-    const { account, rpcPrefs, chainId } = this.props;
+    const { account, network } = this.props;
+    const normalizedAddress = normalizeSafeAddress(account.address);
```

Comment:
This should not be required since `addressSummary` will do it, right?

Reply:
Yes, it does do it in `addressSummary` now